### PR TITLE
Please add this @casey-collab auto-merging

### PR DESCRIPTION
**Pull request by capca**
tnks!

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2021)

### CODE REVIEW

**Comment by capca** [comment]
no able to ser the merging request on my local git log, thanks for answer this!

**Comment by cbuckey-uda** [comment]
@capca I don't understand the problem with your git log. What exactly are you not able to see?

@casey-collab, please review this

File: espanol/noticias/burla-periodista/leones/lanzaronPeriodista.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Y dos de los leones se lanzaron sobre el periodista.
```
Comment (casey-collab) [inline espanol/noticias/burla-periodista/leones/lanzaronPeriodista.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: espanol/noticias/burla-periodista/leones/lanzaronPeriodista.md (hunk 1 above)
Comment (capca) [inline espanol/noticias/burla-periodista/leones/lanzaronPeriodista.md]: Hi, done @casey-collab able to merge now? :)

**Comment by cbuckey-uda** [comment]
@casey-collab, can you take another look?

File: espanol/noticias/burla-periodista/leones/leones.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
 El periodista queda rodeado por 4 expectaculares fieras 
 que podrían destrozrlo tan solo al escuchar a Beatito 
 chasquir sus dedos.
+
+
+Y después:
+
+
+[Se abalanzó](../leones/lanzaronPeriodista.md)
```
Comment (casey-collab) [inline espanol/noticias/burla-periodista/leones/leones.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: espanol/noticias/burla-periodista/leones/lanzaleones/lanzaronPeriodista.md [hunk 3]
```diff
@@ -0,0 +1 @@
+Y dos de los leones se lanzaron sobre el periodista.
```
Comment (casey-collab) [inline espanol/noticias/burla-periodista/leones/lanzaleones/lanzaronPeriodista.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by capca** [comment]
@casey-collab I added the correct link to be able to reach the story from languaje.md

**Comment by casey-collab** [comment]
Looks great! :octocat:

**Pull request merged**